crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -133,6 +145,30 @@ fn D_GGX(roughness: f32, NdotH: f32, h: vec3<f32>) -> f32 {
     return d;
 }
 
+// An approximation of the anisotropic GGX distribution function.
+//
+//                                     1
+//     D(𝐡) = ───────────────────────────────────────────────────
+//            παₜα_b((𝐡 ⋅ 𝐭)² / αₜ²) + (𝐡 ⋅ 𝐛)² / α_b² + (𝐡 ⋅ 𝐧)²)²
+//
+// * `T` = 𝐭 = the tangent direction = the direction of increased roughness.
+//
+// * `B` = 𝐛 = the bitangent direction = the direction of decreased roughness.
+//
+// * `at` = αₜ = the alpha-roughness in the tangent direction.
+//
+// * `ab` = α_b = the alpha-roughness in the bitangent direction.
+//
+// This is from the `KHR_materials_anisotropy` spec:
+// <https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_anisotropy/README.md#image-based-lighting>
```

Comment:
I think this is linking to the wrong section? At least the D_GGX_anistropic function is defined in this section:

```suggestion
// <https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_anisotropy/README.md#individual-lights>
```